lms/djangoapps/edraak_university/forms.py

```diff
@@ -58,3 +71,12 @@ def as_div(self):
             help_text_html=' <span class="helptext">%s</span>',
             errors_on_separate_row=False,
         )
+
+    def clean_full_name(self):
+        return self.cleaned_data['full_name']
+
+    def clean_university_id(self):
+        return self.cleaned_data['university_id']
+
+    def clean_cohort(self):
```

Comment:
- [ ] Don't allow `null` value here as it allowed in the database. All students should submit their cohorts as I understand.